spec/requests/pages_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe "Pages", type: :request do
   describe "GET /home" do
-    it "returns http success" do
+    xit "returns http success" do
```

Comment:
Skipping this test for now. I'm not sure if we'll need to drop it entirely. Might be the case.